books/management/commands/addepub.py

```diff
@@ -249,7 +249,7 @@ def process_epub(self, filename, use_symlink=False):
                                 .split(',')
                             for tag in subject_split:
                                 if tag is not ' ':
-                                    tag = tag.encode("utf-8").lower().strip()
+                                    tag = tag.lower().strip()
```

Comment:
This is the trickiest change - the `israelsailing.epub` file has a tag that is not utf-8, but utf-16, and as a result encoding it to utf-8 causes problems at L256, as taggit seems to try to parse them as ascii and fails.

I'm wondering if you included the line due to problems with tags on other files on your library, but in this case I opted for the least-effort solution, and *assumed that the tags on the epubs won't be in too exotic encodings* (which I'm not really sure it will be the case). It seems we can't assume that they will always be utf8, but I'm not sure either if we can assume they will at least always be unicode. If you could do a test run against some epubs on your library and check the results, or provide more insight about the introducing of the change, that would be ideal: if it mostly works but there is a tiny amount of problematic cases, perhaps the easiest solution would be to enclose L256 in a `try...catch` block and just ignore those tags.

Additionally, adding some epub samples to the `resources/` that have many subjects and help testing other features introduced recently (such as epubs with several authors) would be an excellent idea as well!